Tests/SentryTests/Integrations/Performance/UIViewController/SentryUIViewControllerPerformanceTrackerTests.swift

```diff
@@ -63,8 +63,8 @@ class SentryUIViewControllerPerformanceTrackerTests: XCTestCase {
         clearTestState()
     }
     
-    func testUILifeCycle_ViewDidAppear() {
-        assertUILifeCycle(finishStatus: SentrySpanStatus.ok) { sut, viewController, tracker, callbackExpectation, transactionSpan in
+    func testUILifeCycle_ViewDidAppear() throws {
+        try assertUILifeCycle(finishStatus: SentrySpanStatus.ok) { sut, viewController, tracker, callbackExpectation, transactionSpan in
```

Comment:
`assertUILifeCycle` had some `XCTUnwrap`s in it so had to be marked as `throws`, which bubbled up to a few other test cases.